app/javascript/mastodon/utils/time.tsx

```diff
@@ -0,0 +1,29 @@
+import type { ComponentProps } from 'react';
+
+import { FormattedDate } from 'react-intl';
+
+export const FormattedDateWrapper = (
+  props: ComponentProps<typeof FormattedDate> & { className?: string },
+) => (
+  <FormattedDate {...props}>
+    {(date) => (
+      <time dateTime={tryIsoString(props.value)} className={props.className}>
+        {date}
+      </time>
+    )}
+  </FormattedDate>
+);
+
+const tryIsoString = (date?: string | number | Date): string => {
+  if (!date) {
+    return '';
+  }
+  try {
+    return new Date(date).toISOString();
+  } catch {
+    if (date instanceof Date) {
+      return date.toString();
+    }
```

Comment:
in what case does this fail? if `date` is an instance of Date, then it's a valid arg for the Date constructor.

Reply:
on second thoughts, yes it'd be unnecessary. removed `instanceof` and force-pushed.